app/services/authentication_service.rb

```diff
@@ -19,6 +19,8 @@ def call
 private
 
   def decoded_token
+    return encoded_token if Rails.env.development?
```

Comment:
I think this needs to be

```suggestion
    return { 'email' => encoded_token } if Rails.env.development?
```